android/src/main/java/com/brentvatne/exoplayer/FullScreenPlayerView.kt

```diff
@@ -127,4 +134,25 @@ class FullScreenPlayerView(
         layoutParams.setMargins(0, 0, 0, 0)
         return layoutParams
     }
+
+    private fun navigationBarVisibility() {
```

Comment:
can you rename this function to : updateNavigationBarVisibility please ?

Reply:
For sure.